crates/jstz_node/src/services/operations.rs

```diff
@@ -12,9 +11,23 @@ use tezos_data_encoding::enc::BinWriter;
 use tezos_smart_rollup::inbox::ExternalMessageFrame;
 
 use utoipa_axum::router::OpenApiRouter;
+use utoipa_axum::routes;
 
 pub struct OperationsService;
 
+const OPERATIONS_TAG: &str = "Operations";
+
+/// Inject an operation into Jstz
+#[utoipa::path(
+        post,
+        path = "",
+        tag = OPERATIONS_TAG,
+        responses(
+            (status = 200, description = "Operation successfully injectedd"),
+            (status = 400, description = "Bad request"),
```

Comment:
Do we need these descriptions for 400 and 500 or is this field compulsory? It's fine with these but I think the code itself is clear enough.

Reply:
Removed the descriptions